includes/gateway/class-omise-payment.php

```diff
@@ -418,9 +418,8 @@ protected function payment_failed( $message ) {
 		);
 
 		if ( $this->order() ) {
-			$this->order()->add_order_note(
-				sprintf( __( 'Omise: Payment failed, %s', 'omise' ), $message )
-			);
+			$this->order()->add_order_note( sprintf( __( 'Omise: Payment failed, %s', 'omise' ), $message ) );
```

Comment:
Note, this line is just to remove unnecessary breakline to make it a little more easy to read. Nothing change in term of function.